netebpfext/net_ebpf_ext.c

```diff
@@ -638,27 +700,47 @@ net_ebpf_extension_initialize_wfp_components(_Inout_ void* device_object)
 
     NET_EBPF_EXT_LOG_ENTRY();
 
-    if (_fwp_engine_handle != NULL) {
+    if (_wfp_engine_handle != NULL) {
```

Comment:
please do not change the identifier.

Reply:
I took it to be an accidental typo ('fwp' instead of 'wfp'), so fixed it.  I can certainly revert it back but was curious why you'd prefer the original.